trades/create_channel_sale.js

```diff
@@ -135,7 +147,12 @@ module.exports = ({action, ask, balance, lnd, logger}, cbk) => {
       }],
 
       // Ask for the expiration of the channel sale
-      askForExpiration: ['askForRate', ({}, cbk) => {
+      askForExpiration: ['askForRate', ({askForRate}, cbk) => {
+        // Exit early when using fiat
```

Comment:
I think still ask for expiration but then when the individual invoice is created, create the quote then

Otherwise you can't put a days long sale if it's in fiat?

Reply:
The anchor invoice expiry won't matter in this case, correct?

Reply:
The anchor invoice expiry should reflect the "days" time. When a new trade comes in, it will create a new invoice (the anchor invoice is never shown to the user), that invoice should have the short timer

Maybe it would be easier to just make all of those invoices short time